generatebundlefile/README.md

```diff
@@ -6,53 +6,67 @@ This binary takes in an input file with different fields for curated packages we
 
 Example Input file format
 
+There are 3 types of helm version tags we can input 
+
+1. Exact Name match, a tag named `v1.0.1.1345` will return the sha if there is exact name match.
+2. A substring version with `-latest` at the end of the name. In this case we'll search for the most recent helm chart starting with `v0.1.1` and return that sha.
+3. A tag of `latest` will return the last helm chart pushed to this repo, and it's tag and sha.
+
 ```yaml
-#input.yaml
-kubernetesVersion: "1.21"
+#input_120.yaml
```

Comment:
In general, I think it would be a lot easier not to have examples in the README. They are too hard to maintain.

Reply:
Will remove those as well!